src/main/scala/nl.knaw.dans.easy.validatebag/rules/package.scala

```diff
@@ -31,4 +34,17 @@ package object rules {
         }
     }
   }
+
+  def containsFile(f: Path)(t: TargetBag) = Try {
```

Comment:
why move this file here? Isn't this a `structural` validation? It doesn't make a lot of sense to me to put the method here.

Reply:
Because I am now using it from the bagit package as well.